src/library/scala/collection/immutable/Range.scala

```diff
@@ -435,6 +435,12 @@ sealed abstract class Range(
       }
     }
   }
+
+  override def sorted[B >: Int](implicit ord: Ordering[B]): IndexedSeq[Int] =
+    if (ord eq Ordering.Int)
+      this
```

Comment:
This is not correct for `Range`s with a negative step.

Reply:
Oh, thanks. I'll fix